### PR TITLE
quickstart: Update Tech Detection terminology

### PR DESCRIPTION
**Pull request by kingthorin**
Per:
- https://www.zaproxy.org/addons/
image
- https://www.zaproxy.org/docs/desktop/addons/technology-detection/

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties [hunk 1]
```diff
@@ -58,8 +58,8 @@ quickstart.cmdline.zapit.root.inputs = \tNumber of HTML input fields: {0}
 quickstart.cmdline.zapit.root.links = \tNumber of HTML links: {0}
 quickstart.cmdline.zapit.start = ZAPit scan of {0}
 quickstart.cmdline.zapit.tech = Technology:
-quickstart.cmdline.zapit.tech.disabled = Technology: Not available (enable the wappalyzer add-on)
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:61]: I figured users would see the name more often than the id.

**Review by psiinon — APPROVED** [review]

File: addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (thc202) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:61]: They will need the ID to install from the command line though.

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:61]: Would it be better if I made it "Technology Detection (wappalyzer)" ?

[the remark this answers is not in the document]
Comment (thc202) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:61]: Better would be "Technology Detection (techdetect)" or something like that :)

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:61]: I thought we weren't in a position to change the ID(s)?

[the remark this answers is not in the document]
Comment (thc202) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:61]: We are not yet.

**Pull request merged**